addons/website/static/src/snippets/s_timeline/options.js

```diff
@@ -0,0 +1,34 @@
+odoo.define('website.s_timeline_options', function (require) {
+'use strict';
+
+const options = require('web_editor.snippets.options');
+
+options.registry.Timeline = options.Class.extend({
+    /**
+     * @override
+     */
+    start: function () {
+        var $buttons = this.$el.find('we-button');
+        var $overlayArea = this.$overlay.find('.o_overlay_move_options');
+        $overlayArea.prepend($buttons);
+        $overlayArea.removeClass('d-none');
+        $overlayArea.find('.o_move_handle').addClass('d-none');
```

Comment:
No, I still want the move handle to be able to move it anywhere in the timeline.

Reply:
As agreed . We will leave it like this